### ISSUE TITLE
PDF "Index by grade" add multi-pitch references

### ISSUE THREAD

**Issue by frikiluser** [theCrag/website#4170]
It would be great to have listed the pitches in the "Index by grade" section.

All first pitch may be added.

Further discussion may take place before adding following pitches.

**Comment by frikiluser** [comment]
In case a second pitch has a lower grade than the first one it may induce confusions if the second pitch is listed separately on the "Index by grade" section.

**Comment by lordyavin** [comment]
Dear @frikiluser please provide links, screenshots and mocks of current and desired UI/PDF content.

**Comment by frikiluser** [comment]
Hi @lordyavin !

The route "Canibalismo Vaginal" ( https://www.thecrag.com/en/climbing/spain/mallorca/las-perxes/route/1012386330 ) has two pitches (6a + 7c).

In the pdf version (instant or custom) ( https://static.thecrag.com/pdf/1003799160.pdf ) this route is listed only once in the section named "2. Index by grade". I suggest to add "Canibalismo Vaginal P1" in this section, listed as a 6a route.

I've flashed first pinch of this route and never tried the second pitch. So in my custom pdf I expect the flash icon on the "Canibalismo Vaginal P1".

**Comment by lordyavin** [comment]
Thanks for the details. Now I understand your use case and I agree that the ascent indication in the custom PDF is miss leading. But what you request would mean that the generated PDF contains different structure than the index (at least for the section "Index by grade"). This would increase the complexity of the PDF generator and may not match other users expectations. Looking at the PDF reveals more issues that are more important in my opinion. E.g. I'm missing the number of pitches.

This is not just an issue for the generated PDF it is an issue for the route listing on the website too. One could work around this issue by maintaining each pitch as individual route entry, but I'm not a fan of doing so and if there are already ascents this change would affect others.

Considering the fact that the PDF export feature had low priority in the past years and the man power is limited in general,  I don't expect that such a minor "feature" request will be implemented in the near future. Because of that and the long list of more important issues I ask you to close this issue. If this is really important to you, try to find other community members who back your request and let them comment on or vote for this issue.

**Comment by frikiluser** [comment]
> This is not just an issue for the generated PDF it is an issue for the route listing on the website too.

Yes. I use a lot the custom-pdf, that's why I've noticed it there.

> One could work around this issue by maintaining each pitch as individual route entry, but I'm not a fan of doing so and if there are already ascents this change would affect others.

Yep. That option sucks :-P

> Considering the fact that the PDF export feature had low priority in the past years and the man power is limited in general, I don't expect that such a minor "feature" request will be implemented in the near future. Because of that and the long list of more important issues I ask you to close this issue. If this is really important to you, try to find other community members who back your request and let them comment on or vote for this issue.

OK, got it.

I'll close this issue and hope some one can reopen it on the future.

Thanks for your time @lordyavin !!
